src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -73,17 +79,19 @@
     private GenericContainer<?> cntn;
 
     /**
-     * URL 'http://host:port/my-nuget'.
+     * Server port.
      */
-    private String url;
+    private int port;
 
     @BeforeEach
     void init() throws Exception {
         final String name = "my-nuget";
-        this.server = new ArtipieServer(this.tmp, name, this.config());
-        final int port = this.server.start();
-        this.url = String.format("http://host.testcontainers.internal:%d/my-nuget", port);
-        Testcontainers.exposeHostPorts(port);
+        try (ServerSocket socket = new ServerSocket(0)) {
+            this.port = socket.getLocalPort();
+            this.server = new ArtipieServer(this.tmp, name, this.config());
+        }
+        this.port = this.server.start();
```

Comment:
@genryxy here you are using found free port to put it into config. What I meant is to get the port and start `ArtipieServer` on it. This way you will have correct port in config from the start and will not need to overwrite config later

Reply:
@olegmoz  in this case I need to add new constructor with four params in `ArtipieServer`? Or is there a better solution that I don't see?

Reply:
@genryxy ah, I see. Sorry thought there is option to run `ArtipieServer` on specified port at the moment. Could you please create a puzzle about it? Then we will be good to merge